src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Icons/Given_BitmapIcon.cs

```diff
@@ -46,4 +53,46 @@ public async Task When_Themed_Fluent()
 			await When_Themed();
 		}
 	}
+
+	[TestMethod]
+	public async Task When_Foreground_Set_With_ShowAsMonochrome_False()
+	{
+		var bitmapIcon = new BitmapIcon()
+		{
+			Width = 50,
+			Height = 50,
+			ShowAsMonochrome = false,
+			UriSource = new System.Uri("ms-appx:///Assets/Icons/search.png"),
+			Foreground = new SolidColorBrush(Colors.Green)
+		};
+
+		TestServices.WindowHelper.WindowContent = bitmapIcon;
+		await TestServices.WindowHelper.WaitForLoaded(bitmapIcon);
+		await TestServices.WindowHelper.WaitForIdle();
```

Comment:
The 3 lines can be replaced by `UITestHelper.Load(bitmapIcon)` I think

Reply:
That one is new to me. Most tests use the longer version anyway, so let's leave it at that.

Reply:
If there's a helper that does it better, let's use it to slowly migrate to it the newer tests :)

Reply:
Sure, but I would tread with caution here. The helper explictly calls `UpdateLayout`, which would cover up layouting and invalidation problems.